backend/app/views/spree/admin/option_types/_option_value_fields.html.erb

```diff
@@ -1,11 +1,19 @@
 <tr class="option_value fields" id="spree_<%= dom_id(f.object) %>"  data-hook="option_value">
   <td>
-    <% if f.object.persisted? %>
+    <% if f.object.persisted? && can?(:update_positions, f.object) %>
       <span class="handle"></span>
       <%= f.hidden_field :id %>
     <% end %>
   </td>
-  <td class="name"><%= f.text_field :name %></td>
-  <td class="presentation"><%= f.text_field :presentation %></td>
-  <td class="actions"><%= link_to_remove_fields t('spree.actions.remove'), f, no_text: true %></td>
+  <td class="name">
```

Comment:
Can you keep the previous formatting for these lines? Personally I prefer your formatting, but it just comes down to tastes... I think that we should minimize unnecessary changes, so we keep the git history clean.

Reply:
ok. i thought i do it like in the product property fields for the sake of consistency
https://github.com/solidusio/solidus/blob/master/backend/app/views/spree/admin/product_properties/_product_property_fields.html.erb
But if the git history is more important i'll revert it.

Reply:
ok. done